extensions/amp-wordpress-embed/1.0/amp-wordpress-embed.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseElement} from './base-element';
+import {dict} from '../../../src/core/types/object';
+import {isExperimentOn} from '../../../src/experiments';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-wordpress-embed';
+
+class AmpWordpressEmbed extends BaseElement {
+  /** @override */
+  isLayoutSupported(layout) {
+    userAssert(
+      isExperimentOn(this.win, 'bento') ||
+        isExperimentOn(this.win, 'bento-wordpress-embed'),
+      'expected global "bento" or specific "bento-wordpress-embed" experiment to be enabled'
+    );
+    return super.isLayoutSupported(layout);
+  }
+
+  /** @override */
+  init() {
+    return dict({
+      'requestResize': (height) => {
+        this.forceChangeHeight(height);
```

Comment:
I followed `amp-instagram` bento component and it is using `forceChangeHeight`, so I used this in `amp-wordpress-embed` too. I've just tested with `attemptChangeHeight`, but it is giving me this following warning in console 
```
[CustomElement] Cannot resize element and overflow is not available
```
![Screenshot 2021-06-23 at 11 36 42 AM](https://user-images.githubusercontent.com/1541774/123041562-4a405b00-d417-11eb-87c2-cb33388117e5.png)

I was testing with this code,
```
  init() {
    return dict({
      'requestResize': (height) =>
        this.attemptChangeHeight(height).catch(() => {
          /* ignore failures */
        }),
    });
  }
```

Please suggest what to do.

Reply:
While the overflow is best, should it be required? Namely, I think `amp-wordpress-embed` should be treated the same as `amp-twitter`, `amp-gist`, `amp-facebook`, etc which all will resize regardless of whether it will cause a layout shift.

Reply:
@caroqliu I've updated the code using `attemptChangeHeight`. Please test the component in storybook and let me know what do you think.

Reply:
> While the overflow is best, should it be required? Namely, I think `amp-wordpress-embed` should be treated the same as `amp-twitter`, `amp-gist`, `amp-facebook`, etc which all will resize regardless of whether it will cause a layout shift.

The Bento port is an opportunity to remove legacy layout shift causing behavior whenever possible to provide the most stable end user experience. In design review  yesterday we discussed moving `amp-facebook` to `attemptChangeHeight` (#34969). Note that the Preact version will still resize without user interaction as it is not managed by the AMP runtime.

Reply:
We'll try to address this change in the AMP plugin by capturing the heights of embeds when they are rendered in the editor client-side to then be available server-side when the content is served: https://github.com/ampproject/amp-wp/issues/4729.